client/coral-framework/reducers/user.js

```diff
@@ -30,6 +33,10 @@ export default function user (state = initialState, action) {
   case actions.SAVE_BIO_SUCCESS:
     return state
       .set('settings', action.settings);
+  case actions.RECEIVE_COMMENTS_BY_USER:
+    return state.set('myComments', fromJS(action.comments));
+  case assetActions.RECEIVE_MULTIPLE_ASSETS:
+    return state.set('myAssets', fromJS(action.assets));
```

Comment:
`return state.set('myAssets', action.assets);` 
 `action.comments` instead of using fromJS is not working?